### PR TITLE
fix: make the license key check case insensitive

### PR DESCRIPTION
**Pull request by vikrantgupta25**
#### Screenshots

NA

### CODE REVIEW

File: frontend/src/container/Header/index.tsx [hunk 1]
```diff
@@ -109,9 +110,13 @@ function HeaderContainer(): JSX.Element {
 
 	const { data: licenseData, isFetching, status: licenseStatus } = useLicense();
 
+	const licensesStatus: string =
+		licenseData?.payload?.licenses?.find((e: License) => e.isCurrent)?.status ||
+		'';
+
 	const isLicenseActive =
-		licenseData?.payload?.licenses?.find((e) => e.isCurrent)?.status ===
-		LICENSE_PLAN_STATUS.VALID;
```
Comment (grandwizard28) [inline frontend/src/container/Header/index.tsx:114]: Can we do something like:

`status.toLocalLowerCase() === LICENSE_PLAN_STATUS.VALID.toLocaleLowerCase()` just to ensure that a case insensitive check is happening?

**Review by grandwizard28 — CHANGES_REQUESTED** [review]

File: frontend/src/container/Header/index.tsx (hunk 1 above)
Comment (vikrantgupta25) [reply frontend/src/container/Header/index.tsx:114]: we are doing that in the next line ?

Comment (grandwizard28) [reply frontend/src/container/Header/index.tsx:114]: That was my point, my apologies, I didn't read it correctly.

**Review by grandwizard28 — APPROVED** [review]

**Review by SagarRajput-7 — APPROVED** [review]

**Comment by srikanthccv** [comment]
Next time for new API response payloads do not accept such values where you would have to derive something. Demand the API itself to return the derived result.

**Comment by vikrantgupta25** [comment]
yes did this as a one time thing as there was an incident involved cc :- @grandwizard28

**Review by YounixM — APPROVED** [review]

**Pull request merged**